neos/neos-ui
# FEATURE: Introduce icon configuration for nodetypes

**What I did**
Added a new option `largeIcon` and the node type item component checks if we configure a larger icon and render this instead if the regular icon. The larger icon also use the doubled size.

**How to verify it**

Configure a node type with the property `largeIcon` and check the select node type dialog after that.

```
'Neos.Demo:Content.Text':
  ui:
    icon: 'icon-file-text'
    largeIcon: 'resource://Neos.Demo/Images/customIcon.svg'
```

<img width="1038" alt="Screenshot 2022-02-25 at 18 48 22" src="https://user-images.githubusercontent.com/1014126/155763212-7aaf79dd-2f52-41ae-ba16-b9a06eb9df4b.png">

resolves: #3041
